devlog/_plan/260827_bug_pr_merge_round/021_status_vs_created_at_asymmetry.md

```
@@ -0,0 +1,60 @@
+`status` violate the same contract? — and the honest answer is **yes, it can**.
+
+## Evidence
+
+`tests/server-combo-failover-e2e.test.ts` builds its backup body with
+`responsesSuccess()` (line 216), which already sets `status: "completed"` on both the
+response and the message item. So the byte-exact assertion never exercises the
+`status` backfill — it passes because the field is already present.
+
+Remove that one field from the fixture and the same assertion fails on this branch:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)